### PR TITLE
STG88 GA Release 2023-07-11

### PR DESCRIPTION
**Pull request by ibrahimrabab**
# Description

Updating changelogs and versions for Storage STG88 GA release (2023-07-11).

### CODE REVIEW

**Review by seanmcc-msft — APPROVED** [review]

File: sdk/storage/azure-storage-blob-batch/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
 
-## 12.19.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 12.19.0 (2023-07-11)
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob-batch/CHANGELOG.md:4]: We should put these service version updates under features added for all packages.

```suggestion
### Features Added
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
 
-## 12.22.0-beta.2 (Unreleased)
+## 12.22.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md:6]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
+## 12.23.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob/CHANGELOG.md:6]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-file-datalake/CHANGELOG.md [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
 
-## 12.16.0-beta.2 (Unreleased)
+## 12.16.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
+- Added `DataLakeFileClient.getOutputStream()` which allows users to open an `OutputStream` on the file and write data
+  to it using `ParallelTransferOptions`.
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-file-datalake/CHANGELOG.md:8]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-file-share/CHANGELOG.md [hunk 5]
```diff
@@ -1,14 +1,10 @@
...
+## 12.19.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-file-share/CHANGELOG.md:6]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-internal-avro/CHANGELOG.md [hunk 6]
```diff
@@ -1,14 +1,9 @@
...
+## 12.8.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-internal-avro/CHANGELOG.md:6]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-queue/CHANGELOG.md [hunk 7]
```diff
@@ -1,12 +1,9 @@
...
 
-## 12.18.0-beta.2 (Unreleased)
+## 12.18.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-queue/CHANGELOG.md:6]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

File: sdk/storage/azure-storage-common/CHANGELOG.md [hunk 8]
```diff
@@ -1,16 +1,15 @@
...
 
-## 12.22.0-beta.2 (Unreleased)
+## 12.22.0 (2023-07-11)
 
 ### Features Added
-
-### Breaking Changes
+- Added `ServiceTimeoutPolicy` which allows for service level timeouts to be set on client builders through `HttpPipelinePolicy`. If the
+  server timeout interval elapses before the service has finished processing the request, the service returns an error.
+- Added support for 2023-01-03 service version.
```
Comment (vcolin7) [inline sdk/storage/azure-storage-common/CHANGELOG.md:8]: nit;
```suggestion
- Added support for the `2023-01-03` service version.
```

**Review by vcolin7 — COMMENTED** [review]
Added a few comments on the CHANGELOGs, everything else looks good :)

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**